saos-webapp/src/main/webapp/WEB-INF/view/judgment/judgmentList.jsp

```diff
@@ -63,7 +63,16 @@
 				</span>
 				<ul>
 					<c:forEach items="${judgment.judges}" var="judge" >
-						<li class="judge"><c:out value="${judge.name}" /></li>
+				
+						<li >
+							<span class="judge"><c:out value="${judge.name}" /></span>
+							
+							<c:if test="${judge.presidingJudge}" >
```

Comment:
If we would change `isPresidingJudge()` to `hasRole(JudgeRole)` is it possible to achieve same effect in this statement?
If not then don't change this method signature.
I would also apply this in `judgmentDetails.jsp` (checking some kind of flag instead of iterating over all roles)